### PR TITLE
Fix samples for Event Hubs

### PR DESCRIPTION
**Pull request by ShivangiReja**
## Description

Improved and Fixed all the samples for Event Hubs

### CODE REVIEW

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[44234a6](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/44234a6985f69feac6b9e75f2dfeeaf7624c997e)_: 

:clock10: **Preparing**: average preparing time is _57 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[44234a6](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/44234a6985f69feac6b9e75f2dfeeaf7624c997e)_: 

:clock10: **Incremental building**: average incremental building time is _18 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[44234a6](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/44234a6985f69feac6b9e75f2dfeeaf7624c997e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[client/examples/batchReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/batchSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/createEpochReceiver.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/createEpochReceiver.ts) | :white_check_mark:Succeeded | |
[client/examples/getHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/getHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/iotGetHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/iotGetHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithInteractiveAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithInteractiveAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithSPAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithSPAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSender.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSender.ts) | :white_check_mark:Succeeded | |
[client/examples/streamingReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/streamingReceive.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C12%5Cdef8de6f-0c8d-6946-03eb-eef8a12e22c6%5CPullRequest%5C201902122142102968-194%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=8YFG8rndhtLENKTIKGSxhu6Mb2o2wI9sLdX%2F7wPkf%2FY%3D&st=2019-02-12T21%3A38%3A11Z&se=2019-03-15T21%3A43%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: client/examples/createEpochReceiver.ts [hunk 1]
```diff
@@ -13,33 +13,34 @@ const path = process.env[entityPath] || "";
...
-    console.log("@@@@ receiver with epoch 1.");
     console.log(">>>>> Error occurred for receiver with epoch 1: ", err);
   };
   console.log("$$$$ Will start receiving messages from receiver with epoch value 1...");
-  client.receive("0", onMessage2, onError2, { epoch: 1 });
+  const rcvHandler2 = client.receive(partitionIds[0], onMessage2, onError2, { epoch: 1 });
+  await delay(10000);
+  await rcvHandler2.stop();
+
+  await client.close();
```
Comment (ramya-rao-a) [inline client/examples/createEpochReceiver.ts]: The main difference I see here is that we are stopping the first streaming receiver before starting the second one.

What difference does stopping the receiveHandler make?

File: client/examples/sendReceiveWithInteractiveAuth.ts [hunk 2]
```diff
@@ -17,9 +17,13 @@ async function main(): Promise<void> {
   // azure role assignment create -o contributor --scope /subscriptions/<subscriptionId>/resourceGroups/<rgName>/providers/Microsoft.EventHub/namespaces/<ehNamespaceName> --signInName <user@example.com>
   const credentials = await msrestAzure.interactiveLogin({ tokenAudience: aadEventHubsAudience });
   const client = EventHubClient.createFromAadTokenCredentials(address, path, credentials);
-  await client.send({ body: "Hello awesome world!!" }, "0");
-  const datas = await client.receiveBatch("0", 2, 5, { eventPosition: EventPosition.fromEnqueuedTime(Date.now()) });
-  console.log(">>> EventDataObjects: ", datas);
+  const partitionIds = await client.getPartitionIds();
+  await client.send({ body: "Hello awesome world!!" }, partitionIds[0]);
+  const result: EventData[] = await client.receiveBatch(partitionIds[0], 10, 20, { eventPosition: EventPosition.fromStart() });
```
Comment (ramya-rao-a) [inline client/examples/sendReceiveWithInteractiveAuth.ts]: By using `fromStart`, we will definitely not end up getting the same `Hello awesome world!!` message. I believe that is why we were using `fromEnqueuedTime` before

Comment (ramya-rao-a) [reply client/examples/sendReceiveWithInteractiveAuth.ts]: Same for `sendReceiveWithSPAuth.ts` sample

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[df583e6](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/df583e6e5b136ed35218f842c1e49bb2365da92c)_: 

:clock10: **Pending**: waiting for processors ([8](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **OPS Build is busy**, click [here](https://sitehelp.microsoft.com/) to report any potential performance downgrade issues, or you could view [OPS Portal](https://ops.microsoft.com/#/statistic) for trend of build queue length.

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[df583e6](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/df583e6e5b136ed35218f842c1e49bb2365da92c)_: 

:clock10: **Preparing**: average preparing time is _56 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[df583e6](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/df583e6e5b136ed35218f842c1e49bb2365da92c)_: 

:clock10: **Incremental building**: average incremental building time is _19 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[df583e6](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/df583e6e5b136ed35218f842c1e49bb2365da92c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[client/examples/batchReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/batchSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/createEpochReceiver.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/createEpochReceiver.ts) | :white_check_mark:Succeeded | |
[client/examples/getHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/getHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/iotGetHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/iotGetHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithInteractiveAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithInteractiveAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithSPAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithSPAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSender.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSender.ts) | :white_check_mark:Succeeded | |
[client/examples/streamingReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/streamingReceive.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5Cdef8de6f-0c8d-6946-03eb-eef8a12e22c6%5CPullRequest%5C201902130158301763-194%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=PPv6rGiQxQr%2FaB%2FlUiTWC7F1AqAcm5ltKWV4lZEnLVc%3D&st=2019-02-13T01%3A54%3A29Z&se=2019-03-16T01%3A59%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: client/examples/createEpochReceiver.ts (hunk 1 above)
Comment (ShivangiReja) [reply client/examples/createEpochReceiver.ts]: If we don't stop the receiver and give epoch value 2 to 1st receiver and epoch value 1 to second receiver, we get this error :
image

and we If don't stop the receiver and give epoch value 1 to 1st receiver and epoch value 2 to second receiver, we get this error:
image

If we stop the receivers we don't see these errors.

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[499e4b1](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/499e4b1fbc359ed7a4a88fece2a5466138adda0f)_: 

:clock10: **Preparing**: average preparing time is _54 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[499e4b1](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/499e4b1fbc359ed7a4a88fece2a5466138adda0f)_: 

:clock10: **Incremental building**: average incremental building time is _18 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[499e4b1](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/499e4b1fbc359ed7a4a88fece2a5466138adda0f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[client/examples/batchReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/batchSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/createEpochReceiver.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/createEpochReceiver.ts) | :white_check_mark:Succeeded | |
[client/examples/getHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/getHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/iotGetHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/iotGetHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithInteractiveAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithInteractiveAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithSPAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithSPAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSender.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSender.ts) | :white_check_mark:Succeeded | |
[client/examples/streamingReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/streamingReceive.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5Cdef8de6f-0c8d-6946-03eb-eef8a12e22c6%5CPullRequest%5C201902130241372534-194%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=g4jRo4QB3gM5cIDdDbxajWwqcia8EE6ALujxtMSJ5%2B4%3D&st=2019-02-13T02%3A37%3A39Z&se=2019-03-16T02%3A42%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[81c3ab7](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/81c3ab739c85fb34c98dd3655caecc138f06e450)_: 

:clock10: **Preparing**: average preparing time is _53 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[81c3ab7](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/81c3ab739c85fb34c98dd3655caecc138f06e450)_: 

:clock10: **Incremental building**: average incremental building time is _18 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[81c3ab7](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/81c3ab739c85fb34c98dd3655caecc138f06e450)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[client/examples/batchReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/batchSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/createEpochReceiver.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/createEpochReceiver.ts) | :white_check_mark:Succeeded | |
[client/examples/getHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/getHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/iotGetHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/iotGetHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithInteractiveAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithInteractiveAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithSPAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithSPAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSender.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSender.ts) | :white_check_mark:Succeeded | |
[client/examples/streamingReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/streamingReceive.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5Cdef8de6f-0c8d-6946-03eb-eef8a12e22c6%5CPullRequest%5C201902130257389881-194%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=sAA61uM5HMbYCS%2FvJd1wYbb%2Bj%2FLd%2FSitf4PuJ81xm3A%3D&st=2019-02-13T02%3A53%3A38Z&se=2019-03-16T02%3A58%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[4bc832c](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/4bc832c2ab54eae5e0660bcaae0df4acaf0c3bc1)_: 

:clock10: **Preparing**: average preparing time is _52 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[4bc832c](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/4bc832c2ab54eae5e0660bcaae0df4acaf0c3bc1)_: 

:clock10: **Incremental building**: average incremental building time is _18 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[4bc832c](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/4bc832c2ab54eae5e0660bcaae0df4acaf0c3bc1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[client/examples/batchReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/batchSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/createEpochReceiver.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/createEpochReceiver.ts) | :white_check_mark:Succeeded | |
[client/examples/getHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/getHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/iotGetHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/iotGetHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithInteractiveAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithInteractiveAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithSPAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithSPAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSender.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSender.ts) | :white_check_mark:Succeeded | |
[client/examples/streamingReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/streamingReceive.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5Cdef8de6f-0c8d-6946-03eb-eef8a12e22c6%5CPullRequest%5C201902130300501675-194%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=o4Z%2FUjCPrqeC89XxEPq22kGtt7rygxy4%2FPObh8TwFhE%3D&st=2019-02-13T02%3A56%3A50Z&se=2019-03-16T03%3A01%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[6b9d25a](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/6b9d25a5293fe9f005b7c59bd254c7adb29f0165)_: 

:clock10: **Preparing**: average preparing time is _48 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[6b9d25a](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/6b9d25a5293fe9f005b7c59bd254c7adb29f0165)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/Azure/azure-event-hubs-node, working directory: W:\fhn5-s, target branch: master, target branch commit: 8eb89d186d518eeb97028e771607acf06d5760bc, source branch: 1dd1b9117e274576de149c3bc1226c63/fixSamples, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5Cdef8de6f-0c8d-6946-03eb-eef8a12e22c6%5CPullRequest%5C201902132012523872-194%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=bGOfuCjq5ZeItHVr2MAWYohOI2bS7W5bkLsA26DRl4Q%3D&st=2019-02-13T20%3A08%3A02Z&se=2019-03-16T20%3A13%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[6371487](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/637148739d9368263f5cd3db86e9a85201f3f99a)_: 

:clock10: **Preparing**: average preparing time is _48 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[6371487](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/637148739d9368263f5cd3db86e9a85201f3f99a)_: 

:clock10: **Incremental building**: average incremental building time is _17 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[6371487](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/637148739d9368263f5cd3db86e9a85201f3f99a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[client/examples/batchReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/batchSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/createEpochReceiver.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/createEpochReceiver.ts) | :white_check_mark:Succeeded | |
[client/examples/getHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/getHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/iotGetHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/iotGetHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithInteractiveAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithInteractiveAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithSPAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithSPAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSender.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSender.ts) | :white_check_mark:Succeeded | |
[client/examples/streamingReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/streamingReceive.ts) | :white_check_mark:Succeeded | |
[processor/examples/iothubEph.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/processor/examples/iothubEph.ts) | :white_check_mark:Succeeded | |
[processor/examples/multiEph.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/processor/examples/multiEph.ts) | :white_check_mark:Succeeded | |
[processor/examples/sendBatch.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/processor/examples/sendBatch.ts) | :white_check_mark:Succeeded | |
[processor/examples/singleEph.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/processor/examples/singleEph.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5Cdef8de6f-0c8d-6946-03eb-eef8a12e22c6%5CPullRequest%5C201902132030435179-194%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=tql5N%2FePx%2FrFCFzKvvkHQznHGSfXzrwT5gYrnVYu9is%3D&st=2019-02-13T20%3A26%3A41Z&se=2019-03-16T20%3A31%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by ramya-rao-a** [comment]
@ShivangiReja The build is failing for this PR. Can you take a look?

screen shot 2019-02-13 at 12 37 04 pm

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[616b3b0](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/616b3b08d82248894eea0a147f8259b213c18202)_: 

:clock10: **Preparing**: average preparing time is _48 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[616b3b0](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/616b3b08d82248894eea0a147f8259b213c18202)_: 

:clock10: **Incremental building**: average incremental building time is _17 sec(s)_

**Comment by amarzavery** [comment]
OPS Build status updates of commit _[616b3b0](https://github.com/ShivangiReja/azure-event-hubs-node/pull/194/commits/616b3b08d82248894eea0a147f8259b213c18202)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[client/examples/batchReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/batchSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/batchSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/createEpochReceiver.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/createEpochReceiver.ts) | :white_check_mark:Succeeded | |
[client/examples/getHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/getHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/iotGetHubRuntimeInfo.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/iotGetHubRuntimeInfo.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithInteractiveAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithInteractiveAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/sendReceiveWithSPAuth.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/sendReceiveWithSPAuth.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSendReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSendReceive.ts) | :white_check_mark:Succeeded | |
[client/examples/simpleSender.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/simpleSender.ts) | :white_check_mark:Succeeded | |
[client/examples/streamingReceive.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/client/examples/streamingReceive.ts) | :white_check_mark:Succeeded | |
[processor/examples/iothubEph.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/processor/examples/iothubEph.ts) | :white_check_mark:Succeeded | |
[processor/examples/multiEph.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/processor/examples/multiEph.ts) | :white_check_mark:Succeeded | |
[processor/examples/sendBatch.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/processor/examples/sendBatch.ts) | :white_check_mark:Succeeded | |
[processor/examples/singleEph.ts](https://github.com/ShivangiReja/azure-event-hubs-node/blob/fixSamples/processor/examples/singleEph.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5Cdef8de6f-0c8d-6946-03eb-eef8a12e22c6%5CPullRequest%5C201902132042326077-194%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FU5AYylBFMsIgrZkEXto57ZCZ2eTpnxB35VhJosd3EA%3D&st=2019-02-13T20%3A38%3A30Z&se=2019-03-16T20%3A43%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**